### DISCUSSION TITLE
Draw multiple series

### DISCUSSION THREAD

**Discussion by mahdiye-imanpanah** [Live-Charts/LiveCharts2#1741] [Q&A]
Hi guys
I'm using Livecharts(wpf) to draw a CartesianChart with more than 10 LineSeries.
I have to draw all of the series vertically on top of each other vertically, I read all the samples and documentation in your repository but could not find the right solution.
There was a Mapping property that was the closet option I guess,
this is the way I used it:
` 
Mapping = (logPoint, index) => new(logPoint.X, logPoint.Y + 100 ),
` 
However, when hovering over the chart, the values are the sum of Y and 100.
I want to see the original value not the mapped one.
How can I achieve this goal???

I attached the output to this issue.
![Clipboard - November 24, 2024 3_58 PM](https://github.com/user-attachments/assets/002f55eb-427d-479e-ad18-51ed6b8b2415)

**Comment by beto-rodriguez** [comment]
Hello, it seems that the stacked area sample is what you need:

https://livecharts.dev/docs/WPF/2.0.0-rc4/samples.stackedArea.basic

Is that enough?

**Comment by beto-rodriguez** [comment]
I will close this for now, since this seems to be a question and not an issue with the library, please feel free to reply in the generated discussion, I will help you there.

**Comment by mahdiye-imanpanah** [comment]
Thanks, dear @beto-rodriguez  for your response
But StackedArea is not the answer to my question.
There is a concept in charts named 'Offset'; you can change the chart's start in both the x and y axes.
I could not find anything related to this in your library.
There was just a Mapping option that when I use it the value shown in the tooltip is not correct.
For instance, I have 3 series I want to show in a cartesianChart: 
The values are listed below:
1- [1 , 2 , 3, 4 , 5, 6 , 7, 8]
2- [2 , 5, 6, 8 , 9 , 1 , 4, 8]
3- [-1 , 0 , 3 , 7 , 8 , 0 , 2]

I want to draw these series on top of each other(vertically), after drawing the first, the second chart should start from the highest value in the first item which is 8 in this case, and so on.

Any help would be appreciated.

**Reply by MINT-KISS** [reply]
Are you able to find a way to solve this?
I have a trouble with creating a gantt chart.
Maybe your solution will work for me too.

**Reply by beto-rodriguez** [reply]
Why don't you handle that in your data? @mahdiye-imanpanah, get the max value and build a series with that offset, would that be enough?

I am sorry, I am just trying to understand if this feature is required in the library.

**Reply by mahdiye-imanpanah** [reply]
I did that dear @beto-rodriguez
I tried to handle that by changing the data, but the value shown in the labels is no longer correct when I do.
For instance, when I add 10 to the data {0 , 5 , 10 , 7} the shown value is {10 , 15 , 20 , 17 } instead of {0 , 5 , 10 , 7} 
Let me know if it is not clear to you.
